### DISCUSSION TITLE
How do you get the session id of the user?

### DISCUSSION THREAD

**Discussion by FunctionalMetatable** [ScratchAddons/ScratchAddons#3039] [Help]
```
document.cookie.match(/scratchsessionsid=(.*);/gm)
```
doesnt work

**Comment by WorldLanguages** [comment]
Why do you need the session ID in the first place? You usually do not

**Reply by FunctionalMetatable** [reply]
for the accounts switching addon

**Reply by WorldLanguages** [reply]
That addon is rejected/in pause for now, since it would need us to properly store session IDs and have the user understand the risks
